scipy/spatial/transform/rotation.pyx

```diff
@@ -2169,12 +2171,12 @@ cdef class Rotation(object):
 
         Sample a single rotation:
 
-        >>> R.random(random_state=1234).as_euler('zxy', degrees=True)
+        >>> R.random().as_euler('zxy', degrees=True)  # doctest: +SKIP
         array([-110.5976185 ,   55.32758512,   76.3289269 ])
 
         Sample a stack of rotations:
 
-        >>> R.random(5, random_state=1234).as_euler('zxy', degrees=True)
+        >>> R.random(5).as_euler('zxy', degrees=True)  # doctest: +SKIP
```

Comment:
Why not annotate with `# random`?

Reply:
I was not sure it worked, thanks I will use this then.